src/Containers/OhmmsPETE/OhmmsVector.h

```diff
@@ -155,6 +155,9 @@ class Vector
   ///clear
   inline void clear() { nLocal = 0; }
 
+  ///zero
+  inline void zero() { std::fill_n(X, nAllocated, 0); }
```

Comment:
Feel better to put `T()` instead of `0`.

Reply:
me too